packages/core/src/di/injector_compatibility.ts

```diff
@@ -110,9 +113,11 @@ export function injectInjectorOnly<T>(
  * @see inject
  * @codeGenApi
  */
-export function ɵɵinject<T>(token: Type<T>|InjectionToken<T>): T;
-export function ɵɵinject<T>(token: Type<T>|InjectionToken<T>, flags?: InjectFlags): T|null;
-export function ɵɵinject<T>(token: Type<T>|InjectionToken<T>, flags = InjectFlags.Default): T|null {
+export function ɵɵinject<T>(token: Type<T>|InjectionToken<T>|AbstractType<T>): T;
```

Comment:
It's just a nitpick, but I would suggest reordering `AbstractType` and `InjectionToken`:
```suggestion
export function ɵɵinject<T>(token: Type<T>|AbstractType<T>|InjectionToken<T>): T;
```
as they're two sides of the same coin, vs `InjectionToken` which is more of a foreign currency ;)